validator/Dockerfile

```diff
@@ -0,0 +1,36 @@
+# Copyright 2020 Paolo Smiraglia <paolo.smiraglia@gmail.com>
```

Comment:
Why me? It's your... :smirk:

Reply:
Well, I used the LICENCE file from the repository root... I mean, I produced the code, but it's in your repo after all? 😅 🤷🏻‍♂️

Reply:
Yep, but the code is your...